addons/website/views/snippets/s_numbers_showcase.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_showcase" name="Numbers Showcase">
+    <section class="s_numbers_showcase pt80 pb80 o_colored_level">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="8" style="gap: 24px;">
+                <div class="o_grid_item g-height-6 g-col-lg-5 col-lg-5 justify-content-start" style="--grid-item-padding-y: 32px; --grid-item-padding-x: 32px; grid-area: 1 / 1 / 9 / 7; z-index: 1;">
```

Comment:
edition glitch: `grid-area` or `col-` classes are not set correctly 😉 

https://github.com/user-attachments/assets/c2c80411-80ff-4e11-a915-5e165e6dd9e8